frontend/preview/img/favicon.png

```diff
@@ -0,0 +1 @@
+../../img/favicon.png
```

Comment:
Why do some html pages say the favicon is `favicon.ico` and others say it is `img/favicon.png`?

Reply:
Fixed, using img/favicon.png everywhere now.

Reply:
apologies, I'm merging this now and resolving myself. let me know if that offends.